cmd/sqlite3def/sqlite3def_test.go

```diff
@@ -143,6 +143,20 @@ func TestSQLite3defConfigIncludesSkipTables(t *testing.T) {
 	assertEquals(t, apply, nothingModified)
 }
 
+func TestSQLite3defBlobType(t *testing.T) {
```

Comment:
Can you test this in `TestApply`? All the tests other than `TestApply` should only test CLI features, and SQL manipulation should be tested in `cmd/sqlite3def/tests.yml` because it's faster and less redundant.